rclpy/rclpy/time.py

```diff
@@ -44,6 +44,10 @@ def __init__(
     def nanoseconds(self):
         return self._time_handle.nanoseconds
 
+    @property
+    def seconds(self):
+        return self.nanoseconds / CONVERSION_CONSTANT
+
```

Comment:
It looks like `seconds_nanoseconds()` returns the number of seconds as an integer, while this PR's `t.seconds` returns seconds as a float. If we had all three methods, it would look a little odd to me because `(t.seconds, t.nanoseconds) != t.seconds_nanoseconds()`.

If there's a desire to get floating point seconds since the epoch, maybe implementing it as `def __float__(self):` would be clearer?

```python
t = rclpy.time.Time(...)
secons_since_epoch = float(t)
```

Reply:
Sorry I opened this PR and disappeared. Thanks for the comments, I'll try to address your comments as soon as I find some time again.